File: app/Quotes/bs/src/pages/bye.ml
(*  Copyright 22-Jul-2018 ºDeme
    GNU General Public License - V3 <http://www.gnu.org/licenses/>
*)

open Main
open Ui
open I18n

let show () =
  let w = q "div" [][
    q "div" [Klass "title"; Html {j|&nbsp;<br>$app<br>&nbsp;|j}][];
    q "div" [][
      q "table" [
          Klass "border";
          Att ("width", "100%");
          Style "background-color: #f8f8f8;border-collapse: collapse;"
        ][
        q "tr" [][
          q "td" [
            Style "padding:0px 10px 0px 10px;";
            Html (i18f (i18 "Logout-message") [app])][]]]]]
  in
  show_root w

File: app/MarketLeague/js/src/core/Bye.js
// Copyright 18-Jul-2019 ºDeme
// GNU General Public License - V3 <http://www.gnu.org/licenses/>

import Main from "../Main.js";
import {_args, _} from "../I18n.js";
import Ui from "../dmjs/Ui.js";

const $ = Ui.$;
/** Bye page. */
export default class Bye {
  /**
   * @param {!Main} main Main page
   */
  constructor (main) {
    /**
     * @private
     * @type {!Main}
     */
    this._main = main;
  }

  /**
   * @return {void}
   */
  show () {
    this._main.view.removeAll().add(
      $("div")
        .add($("div").klass("title")
          .html(`&nbsp;<br>${Main.app}<br>&nbsp;`))
        .add($("div")
          .add($("table")
            .att("class", "border")
            .att("width", "100%")
            .att("style",
              "background-color: #f8f8f8;" +
              "border-collapse: collapse;")
            .add($("tr")
              .add($("td")
                .att("style", "padding:0px 10px 0px 10px;")
                .html(_args(_("Logout-message"), Main.app))))))
    );
  }
}
